lib/bridge.js

```diff
@@ -88,6 +88,10 @@ var INTENT_CULL_EVICT_AFTER_MS = 1000 * 60 * 15; // 15 minutes
  * parameters in {@link Bridge~onEvent}. Disabling the context makes the
  * bridge do fewer database lookups, but prevents there from being a
  * <code>context</code> parameter. Default: false.
+ * @param {string=} opts.groupLocalpart If supplied, users and rooms will be associated
+ * with the group. If the group does not exist, a default one will be created prior to
```

Comment:
Worth mentioning at which point the users and rooms will be associated, I think. Also worth mentioning that they won't be unassociated and that users are invited whereas rooms are just associated.

Reply:
Actually, you might want to describe it totally differently and only in scope of the as-bridge library itself. AFAIC, the groupLocalpart causes three things to happen:
 - Creates the group when it is first required
 - Automatically joins ghost users to the group
 - Adds `m.room.related_rooms` event to rooms created via alias request with the group's ID in it

Reply:
This morning I was also thinking of changing it from a single string to an object, just to be able to provide a few more options. Namely for the future case of being able to control membership visibility. At this point it'd still just be the localpart though.

Reply:
yep, sounds like something I should've picked up on, actually. WFM